src/wirecloud/platform/iwidget/models.py

```diff
@@ -22,7 +22,7 @@
 from django.utils.translation import ugettext as _
 
 from wirecloud.commons.fields import JSONField
-from wirecloud.platform.wiring.utils import remove_related_iwidget_connections
```

Comment:
Take into account that this function is not used from a class, please, provide a more descriptive name (e.g. `remove_widget_from_wiring_status` or `remove_related_widget_connections`)